config/manager/manager.yaml

```diff
@@ -19,6 +19,7 @@ spec:
         control-plane: tf-controller
     spec:
       terminationGracePeriodSeconds: 10
+      serviceAccountName: controller
```

Comment:
Ouch my bad, sorry @phoban01 
I ran `kustomize build ./config/manager > ./config/release/tf-controller.deployment.yaml` to double check,
and found that only this one needs to be `tf-controller`. The prefix does not kick in here.

Reply:
other than that it's LGTM now.

Reply:
Thanks, fixed now @chanwit